ultralytics/yolo/engine/model.py

```diff
@@ -191,6 +191,9 @@ def train(self, **kwargs):
             self.trainer.model = self.trainer.get_model(weights=self.model if self.ckpt else None, cfg=self.model.yaml)
             self.model = self.trainer.model
         self.trainer.train()
+        # update model and configs after training
+        self.model, _ = attempt_load_one_weight(str(self.trainer.best))
+        self.overrides = self.model.args
```

Comment:
@Laughing-q what's the difference bw doing this and directly setting `self.model = self.trainer.model`? Are both same

Reply:
our export needs `pt_path` for file name which we give it to model in `attempt_load_one_weight`.

Reply:
Okay got it